src/gluonts/nursery/robust-mts-attack/pts/modules/gaussian_diffusion.py

```diff
@@ -43,8 +43,7 @@ def noise_like(shape, device, repeat=False):
 
 def cosine_beta_schedule(timesteps, s=0.008):
     """
-    cosine schedule
-    as proposed in https://openreview.net/forum?id=-NEXDKk8gZ
+    cosine schedule as proposed in https://openreview.net/forum?id=-NEXDKk8gZ.
```

Comment:
```suggestion
    Cosine schedule as proposed in https://openreview.net/forum?id=-NEXDKk8gZ.
```

Reply:
I think it is docs required to start with Capital and ends with .